sdk/firmware.ldscript.in

```diff
@@ -234,8 +242,10 @@ SECTIONS
 
 		@software_revoker_header@
 
-		# Magic number
-		LONG(0x43f6af90);
+		# Magic number.
+		# New versions of this can be generated with:
+		# head /dev/random | shasum | cut -c 0-8
```

Comment:
While you're adding commentary, xref `sdk/core/loader/types.h:/is_magic_valid`?

Reply:
Done.